PDF/basic.py

```diff
@@ -1,34 +1,20 @@
 from fpdf import FPDF
 
-# Author: @NavonilDas
```

Comment:
Only this is what to be removed.
Even as optional.
But, as we are approaching a standardized open sourced, so removing this may be legitimate.
But, removing the comments which make the program better is not good.
Comments and documentations are the essence as they help in understanding the program much more better.